scripts/diagnose.sh

```diff
@@ -126,7 +126,7 @@ commands=(
 	'echo === SUPERVISOR ==='
 	'$ENG exec resin_supervisor cat /etc/resolv.conf'
 	'$ENG logs resin_supervisor'
-	'curl --max-time 5 localhost:48484/v1/healthy'
+	'curl --max-time 5 localhost:'"${LISTEN_PORT}"'/v1/healthy'
```

Comment:
can you please clarify the reason for the single quotes around the double quotes?

Reply:
These quotes are required to prevent possible variable splitting/globbing, as per https://github.com/koalaman/shellcheck/wiki/SC2206